tac/gui/dashboards/base.py

```diff
@@ -42,7 +42,7 @@ def stop(self):
 
 def start_visdom_server() -> subprocess.Popen:
     """Start the visdom server."""
-    visdom_server_args = ["python", "-m", "visdom.server", "-env_path", os.path.join(CUR_DIR, ".visdom_env")]
+    visdom_server_args = ["python", "-m", "visdom.server", "-env_path", os.path.join(CUR_DIR, "..", ".visdom_env")]
```

Comment:
Update the relative path to `.visdom_env` to comply with the new package structure.